.github/workflows/workflow-call.yaml

```diff
@@ -0,0 +1,91 @@
+name: Replay Workflow Call
+
+on:
```

Comment:
Are you calling this from somewehre? Any reason this is not a reusable action like the ones in .github/actions?